logstash-output-pagerduty.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-output-pagerduty'
-  s.version         = '1.0.0'
+  s.version         = '1.1.0'
```

Comment:
I don't think its a controversy, the rules are more are we introducing breaking changes? If no, it should be a patch level.